.github/workflows/groovy-build-test.yml

```diff
@@ -35,13 +35,11 @@ jobs:
       - uses: actions/setup-java@v4
         with:
           distribution: 'zulu'
-          java-version: |
-            ${{ matrix.java }}
-            17
```

Comment:
This was designed to provide Java from the matrix list plus Java 17 for the main build.  It already feeds Java 2x to the tests on line 44.